### ISSUE TITLE
Client crashes on startup

### ISSUE THREAD

**Issue by ppratscher** [pipermerriam/ethereum-alarm-client#8]
I was playing around with the client but it instantly crashes with the following error:

```
username@Ubuntu-1404-trusty-64-minimal:~/alarm-clock-client/env/bin$ ./eth_alarm scheduler --client rpc --address 0x1a3894096e11728107deb3fe0fb18624753ad793
BLOCKSAGE: INFO: 2016-01-31 16:53:13,698 Starting block sage
BLOCKSAGE: INFO: 2016-01-31 16:53:13,710 > Heartbeat: block #932940 : block_time: 10.0
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/username/alarm-clock-client/env/local/lib/python2.7/site-packages/eth_alarm_client/scheduler.py", line 70, in monitor
    self.schedule_calls()
  File "/home/username/alarm-clock-client/env/local/lib/python2.7/site-packages/eth_alarm_client/scheduler.py", line 178, in schedule_calls
    max(0, self.block_sage.current_block_number - self.minimum_grace_period),
  File "/home/username/alarm-clock-client/env/local/lib/python2.7/site-packages/eth_alarm_client/utils.py", line 22, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/username/alarm-clock-client/env/local/lib/python2.7/site-packages/eth_alarm_client/scheduler.py", line 57, in minimum_grace_period
    return self.scheduler.getMinimumGracePeriod()
  File "/home/username/alarm-clock-client/env/local/lib/python2.7/site-packages/populus/contracts/functions.py", line 57, in __call__
    return self.call(*args, **kwargs)
  File "/home/username/alarm-clock-client/env/local/lib/python2.7/site-packages/populus/contracts/functions.py", line 91, in call
    return self.cast_return_data(output)
  File "/home/username/alarm-clock-client/env/local/lib/python2.7/site-packages/populus/contracts/common.py", line 80, in cast_return_data
    raise EmptyDataError("call to {0} unexpectedly returned no data".format(self))
EmptyDataError: call to getMinimumGracePeriod() unexpectedly returned no data
```

geth is running with rpc enable at port 8545 and with the account unlocked.

**Comment by pipermerriam** [comment]
This error is indicative of running the client pointed at an ethereum address that doesn't contain code.  This can happen if you were running the client against a test chain for example.  If you are running this on the morden network then you should add `--address 0x26416b12610d26fd31d227456e9009270574038f`.  The client defaults to the live address.

**Comment by pipermerriam** [comment]
I just pushed some new code to master and released as `7.0.0b8` which does a check at startup to sanity check that the address being used for the scheduler is correct

**Comment by pipermerriam** [comment]
Let me know if you're still having problems.
